### PR TITLE
Recolored cell backgrounds based on new specification.

### PR DESCRIPTION
**Pull request by bobdel**
I just did the minimum code changes to accomplish this task. I am unsure if I handled the Double in the best way possible. I chose rounding because the use case here does not need that level of precision.

### CODE REVIEW

File: ParkenDD/Colors.swift [hunk 1]
```diff
@@ -12,30 +12,30 @@ struct Colors {
...
 	- parameter percentage: value between 0 and 1
-	- parameter emptyLots: number of empty lots
 
 	- returns: UIColor
 	*/
-	static func colorBasedOnPercentage(_ percentage: Double, emptyLots: Int) -> UIColor {
+	static func colorBasedOnPercentage(_ percentage: Double) -> UIColor {
 
-		let hue = 1 - (percentage * 0.3 + 0.7) // I want to limit this to the colors between 0 and 0.3
+        // Normalize Double from undocumented API
```
Comment (olivaresf) [inline ParkenDD/Colors.swift:30]: Don't need this comment. Remove it.

File: ParkenDD/Colors.swift [hunk 2]
```diff
@@ -12,30 +12,30 @@ struct Colors {
...
-
-		if emptyLots <= 0 {
-			return UIColor(hue: CGFloat(hue), saturation: 0.54, brightness: 0.7, alpha: 1.0)
-		}
-		return UIColor(hue: CGFloat(hue), saturation: 0.54, brightness: 0.8, alpha: 1.0)
+        switch normalizedPercentage {
+        case 85...99: return freeSpaceLimited
+        case 40...84: return freeSpaceMedium
+        case 1...39: return freeSpaceHighest
+        default: return freeSpaceNone
```
Comment (olivaresf) [inline ParkenDD/Colors.swift:36]: I like the switch because it's a catch all for invalid inputs. Great job 👍

File: ParkenDD/Colors.swift (hunk 1 above)
Comment (olivaresf) [inline ParkenDD/Colors.swift]: Unnecessary comment. Please remove.

**Review by olivaresf — CHANGES_REQUESTED** [review]
This is a difficult lesson to learn, but I think you did really well. I like that you didn't touch anything else. Live code, like the one in the project, is always messy and imperfect so there will _always_ be something to improve. It's important that we, as developers, provide value by fixing issues and creating features in a limited scope.

Great job 😄

File: ParkenDD/Colors.swift (hunk 1 above)
Comment (bobdel) [reply ParkenDD/Colors.swift]: Removed comment.

**Pull request merged**